util/cloud/s3.cc

```diff
@@ -220,6 +238,20 @@ error_code S3Bucket::ListObjects(string_view path, ListObjectCb cb) {
   return xml::ParseListObj(xml, std::move(cb));
 }
 
+error_code S3Bucket::ListAllObjects(string_view bucket_path, ListObjectCb cb) {
```

Comment:
1. Decide on naming (List vs ListAll, Get vs List)
2. Maybe pass batch size as optional param

Reply:
are those comments to you? to me? :)

Reply:
To you 😅 Imagine a question mark after each of them

Reply:
1. Naming is fine (I think List is better here).
2. No need, sane defaults should work until they won't and then we will decide what to do.